['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpeklo2n0f/solution_0.py", line 46, in <module>\n    print(Solution().canSortArray(nums = [14,7,3,1,12]))\nTypeError: canSortArray() got an unexpected keyword argument \'nums\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpeklo2n0f/solution_1.py", line 46, in <module>\n    print(Solution().canSortArray(nums = [2,2,2,2,2]))\nTypeError: canSortArray() got an unexpected keyword argument \'nums\'\n']